mantidproject/mantid
# A tool for drawing arbitrary masks

Fixes #12808.

**To test**
1. Open the instrument view and got to the Draw tab.
2. Use the tool button with a paint brush to draw a shape. Left button - draw, right button - erase.
3. Check that it works as any other shape.
4. Check that the shape works in the Pick tab as well.
